app/services/forecast_service.js

```diff
@@ -20,7 +20,7 @@ class ForecastService {
     })
     .catch((error) => {
       console.log({error:error, location:'getGeInfo'});
-      return {error_message: error};
+      return {error_message: error.message};
```

Comment:
Nice use of sending the error.message. That makes it much easier for the user to understand the error messages.